circle.yml

```diff
@@ -14,11 +14,17 @@ dependencies:
   post:
     - ./node_modules/.bin/gulp lint
     - ./node_modules/.bin/webdriver-manager update
-    - ./node_modules/.bin/webdriver-manager start:
-          background: true
+    - wget http://selenium-release.storage.googleapis.com/3.0-beta4/selenium-server-standalone-3.0.0-beta4.jar -P ./node_modules/webdriver-manager/selenium
```

Comment:
Note to self: Can't use 3.0.0 or 3.0.1, as these both have a bug that breaks firefox.